frontend/src/features/apportionment/components/Apportionment.stories.tsx

```diff
@@ -0,0 +1,83 @@
+import { Story } from "@ladle/react";
+
+import { PoliticalGroup } from "@/api";
+
+import * as equalOrMore from "../testing/19-or-more-seats";
+import * as lessThan from "../testing/less-than-19-seats";
```

Comment:
```suggestion
import * as lt19Seats from "../testing/less-than-19-seats";
```
I used this in ApportionmentResidualSeatsPage.test.tsx in https://github.com/kiesraad/abacus/pull/1239